arbor/include/arbor/common_types.hpp

```diff
@@ -97,6 +97,28 @@ struct cell_global_label_type {
     cell_global_label_type(cell_gid_type gid, cell_tag_type tag, lid_selection_policy policy): gid(gid), label(std::move(tag), policy) {}
 };
 
+// User facing handle for referring to probes and similar
+
+struct cell_address_type {
```

Comment:
This could also be added to the API docs.

Reply:
👍